src/components/tutor-input.cjsx

```diff
@@ -47,15 +48,12 @@ TutorInput = React.createClass
   forwardLabelClick: -> @focus()
 
   render: ->
-    classes = ['form-control']
-    wrapperClasses = ["form-control-wrapper", "tutor-input"]
-    wrapperClasses.push(@props.className) if @props.className
+    classes = classnames 'form-control', @props.class,
```

Comment:
Might be too much to change but should `TutorInput` take `className` as a prop instead of `class`?

/ cc @jivewise